Error when running testUpdateHostsFile.py

I wonder why I got this error when running `testUpdateHostsFile.py` 
`updateHostsFile.py` run just fine

I already installed the dependencies with pip. My python version is 3.8.2

```
PS C:\Users\blue\Utility\hosts-master> python .\testUpdateHostsFile.py
..............................................................................................F.FF............
======================================================================
FAIL: test_add_fields (__main__.TestUpdateReadmeData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".\testUpdateHostsFile.py", line 1136, in test_add_fields
    self.assertEqual(actual, expected)
AssertionError: {'foo': 'bar', 'base': {'location': 'foo/',[34 chars]ts'}} != {'base': {'location': 'foo\\', 'sourcesdata[35 chars]bar'}
- {'base': {'entries': 5, 'location': 'foo/', 'sourcesdata': 'hosts'},
?                                         ^

+ {'base': {'entries': 5, 'location': 'foo\\', 'sourcesdata': 'hosts'},
?                                         ^^

   'foo': 'bar'}

======================================================================
FAIL: test_modify_fields (__main__.TestUpdateReadmeData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".\testUpdateHostsFile.py", line 1153, in test_modify_fields
    self.assertEqual(actual, expected)
AssertionError: {'base': {'location': 'foo/', 'entries': 5, 'sourcesdata': 'hosts'}} != {'base': {'location': 'foo\\', 'sourcesdata': 'hosts', 'entries': 5}}
- {'base': {'entries': 5, 'location': 'foo/', 'sourcesdata': 'hosts'}}
?                                         ^

+ {'base': {'entries': 5, 'location': 'foo\\', 'sourcesdata': 'hosts'}}
?                                         ^^


======================================================================
FAIL: test_set_extensions (__main__.TestUpdateReadmeData)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".\testUpdateHostsFile.py", line 1177, in test_set_extensions
    self.assertEqual(actual, expected)
AssertionError: {'com-org': {'location': 'foo/', 'entries': 5, 'sourcesdata': 'hosts'}} != {'com-org': {'location': 'foo\\', 'sourcesdata': 'hosts', 'entries': 5}}
- {'com-org': {'entries': 5, 'location': 'foo/', 'sourcesdata': 'hosts'}}
?                                            ^

+ {'com-org': {'entries': 5, 'location': 'foo\\', 'sourcesdata': 'hosts'}}
?                                            ^^


----------------------------------------------------------------------
Ran 110 tests in 0.118s

FAILED (failures=3)
```

--- a/testUpdateHostsFile.py
+++ b/testUpdateHostsFile.py
@@ -64,7 +64,7 @@ class Base(unittest.TestCase):
     @property
     def sep(self):
         if platform.system().lower() == "windows":
-            return "\\"
+            return "/"
         return os.sep
 
     def assert_called_once(self, mock_method):

--- a/testUpdateHostsFile.py
+++ b/testUpdateHostsFile.py
@@ -64,7 +64,7 @@ class Base(unittest.TestCase):
     @property
     def sep(self):
         if platform.system().lower() == "windows":
-            return "/"
+            return "\\"
         return os.sep
 
     def assert_called_once(self, mock_method):
@@ -1125,12 +1125,13 @@ class TestUpdateReadmeData(BaseMockDir):
         )
         update_readme_data(self.readme_file, **kwargs)
 
+        if platform.system().lower() == "windows":
+            sep = "/"
+        else:
+            sep = self.sep
+
         expected = {
-            "base": {
-                "location": "foo" + self.sep,
-                "sourcesdata": "hosts",
-                "entries": 5,
-            },
+            "base": {"location": "foo" + sep, "sourcesdata": "hosts", "entries": 5},
             "foo": "bar",
         }
 
@@ -1147,8 +1148,13 @@ class TestUpdateReadmeData(BaseMockDir):
         )
         update_readme_data(self.readme_file, **kwargs)
 
+        if platform.system().lower() == "windows":
+            sep = "/"
+        else:
+            sep = self.sep
+
         expected = {
-            "base": {"location": "foo" + self.sep, "sourcesdata": "hosts", "entries": 5}
+            "base": {"location": "foo" + sep, "sourcesdata": "hosts", "entries": 5}
         }
 
         with open(self.readme_file, "r") as f:
@@ -1167,12 +1173,13 @@ class TestUpdateReadmeData(BaseMockDir):
         )
         update_readme_data(self.readme_file, **kwargs)
 
+        if platform.system().lower() == "windows":
+            sep = "/"
+        else:
+            sep = self.sep
+
         expected = {
-            "com-org": {
-                "location": "foo" + self.sep,
-                "sourcesdata": "hosts",
-                "entries": 5,
-            }
+            "com-org": {"location": "foo" + sep, "sourcesdata": "hosts", "entries": 5}
         }
 
         with open(self.readme_file, "r") as f:

--- a/testUpdateHostsFile.py
+++ b/testUpdateHostsFile.py
@@ -8,6 +8,7 @@
 import json
 import locale
 import os
+import platform
 import re
 import shutil
 import sys
@@ -62,6 +63,8 @@ class Base(unittest.TestCase):
 
     @property
     def sep(self):
+        if platform.system().lower() == "windows":
+            return "\\"
         return os.sep
 
     def assert_called_once(self, mock_method):

